src/test/values_schemas/03_generate_schema/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+# yaml-language-server: $schema=./values.schema.json
+
+domain: "###ZARF_VAR_DOMAIN###"
+
+global:
+  kubectl:
+    image:
+      repository: set-by-flavor
+      tag: set-by-flavor
+
+botAccounts:
+  enabled: false
+  # accounts:
+  #   - username: renovatebot
+  #     scopes:
+  #       - api
+  #       - read_repository
+  #       - write_repository
+  #     secret:
+  #       name: gitlab-renovatebot
+  #       namespace: renovate
+  #       keyName: TOKEN
+
+settingsJob:
+  enabled: true
+  schedule: "0 2 * * *"  # Run at 2:00 AM every day
+  # Settings here come from: https://docs.gitlab.com/ee/api/settings.html
+  # NOTE: These keys are converted to a JSON object that is converted to query
+  # parameters later on so any nested objects/arrays should be encoded as a
+  # string query parameter instead of their full type.
+  application:
+    default_snippet_visibility: private
```

Comment:
we will need to figure out how to handle additionalProperties on these nested objects - this is not all of the options for GitLab in this case and it shouldn't fail if a user adds an additional property here